src/common/module_builder.h

```diff
@@ -51,6 +53,10 @@ class ModuleBuilder {
   void
   createFlatbufferBinary(const mlir::OwningOpRef<mlir::ModuleOp> &mlir_module);
 
+  // Fills up the m_is_output_scalar array with information is the output type
+  // scalar or not.
+  void collectOutputTypes(mlir::ModuleOp &&module);
+
   // Prints module to console for debug purposes.
   static void print_module(mlir::OwningOpRef<mlir::ModuleOp> &mlir_module);
```

Comment:
Change this to camelCase as well if it is not used in too many places and will bloat PR.